ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -29,13 +29,13 @@ import UIKit
 }
 
 /// View that represents the button.
-public struct MSFButtonView: View {
+public struct FluentButton: View {
```

Comment:
Did we want to rename this file to `FluentButton.swift` as well? I just realized we didn't discuss this earlier. (I don't have any strong opinion)

Reply:
I think we can keep this file name as we're using the core "Button" name throughout the other files, including the ButtonModifiers, and the MSFButton everywhere for the objc compatible objects and YAML file too (AP_MSFButtonTokens and its sub APs).